checks.d/mongo.py

```diff
@@ -61,6 +58,84 @@ class MongoDb(AgentCheck):
         "metrics.repl.buffer.count",
         "metrics.repl.buffer.maxSizeBytes",
         "metrics.repl.buffer.sizeBytes",
+
+        # mongo 3.x metrics
+        "locks.Collection.acquireCount.R",
```

Comment:
The metrics you're reporting throughout don't quite add up with what I see. It could be a difference between MMAP and WiredTiger (I'm running WiredTiger and am guessing you did this with MMAP?)

Ones I see that you're missing:

    "locks.Collection.acquireCount.r",
    "locks.Collection.acquireCount.w",
    "locks.Database.acquireCount.R",
    "locks.Database.acquireCount.W",
    "locks.Database.acquireWaitCount.R",
    "locks.Database.acquireWaitCount.W",
    "locks.Database.timeAcquiringMicros.R",
    "locks.Database.timeAcquiringMicros.W",
    "locks.Global.acquireCount.R",
    "locks.Global.acquireCount.W",
    "locks.Global.acquireWaitCount.R",
    "locks.Global.acquireWaitCount.W",
    "locks.Global.timeAcquiringMicros.R",
    "locks.Global.timeAcquiringMicros.W",

    "network.bytesIn,
    "network.bytesOut",
    "network.numRequests",

    "metrics.commands._isSelf.failed",
    "metrics.commands._isSelf.total",
    "metrics.commands.aggregate.failed",
    "metrics.commands.aggregate.total",
    "metrics.commands.buildInfo.failed",
    "metrics.commands.buildInfo.total",
    "metrics.commands.collStats.failed",
    "metrics.commands.collStats.total",
    "metrics.commands.dbStats.failed",
    "metrics.commands.dbStats.total",
    "metrics.commands.distinct.failed",
    "metrics.commands.distinct.total",
    "metrics.commands.getLastError.failed",
    "metrics.commands.getLastError.total",
    "metrics.commands.getLog.failed",
    "metrics.commands.getLog.total",
    "metrics.commands.getnonce.failed",
    "metrics.commands.getnonce.total",
    "metrics.commands.isMaster.failed",
    "metrics.commands.isMaster.total",
    "metrics.commands.listCollections.failed",
    "metrics.commands.listCollections.total",
    "metrics.commands.listIndexes.failed",
    "metrics.commands.listIndexes.total",
    "metrics.commands.ping.failed",
    "metrics.commands.ping.total",
    "metrics.commands.renameCollection.failed",
    "metrics.commands.renameCollection.total",
    "metrics.commands.replSetGetStatus.failed",
    "metrics.commands.replSetGetStatus.total",
    "metrics.commands.replSetHeartbeat.failed",
    "metrics.commands.replSetHeartbeat.total",
    "metrics.commands.replSetUpdatePosition.failed",
    "metrics.commands.replSetUpdatePosition.total",
    "metrics.commands.saslContinue.failed",
    "metrics.commands.saslContinue.total",
    "metrics.commands.whatsmyuri.failed",
    "metrics.commands.whatsmyuri.total",
    "metrics.operation.writeConflicts",
    "metrics.repl.buffer.count",
    "metrics.repl.buffer.maxSizeBytes",
    "metrics.repl.buffer.sizeBytes",
    "metrics.repl.preload.docs.num",
    "metrics.repl.preload.docs.totalMillis",
    "metrics.repl.storage.freelist.search.bucketExhausted",
    "metrics.repl.storage.freelist.search.requests",
    "metrics.repl.storage.freelist.search.scanned",